plugins/docker-options/commands

```diff
@@ -4,12 +4,15 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 
 case "$1" in
   help | docker-options:help)
-    cat<<EOF
-    docker-options <app>, Display app's Docker options for all phases
-    docker-options <app> <phase(s)>, Display app's Docker options for phase (comma separated phase list)
+    help_content='    docker-options <app>, Display app'"'"'s Docker options for all phases
+    docker-options <app> <phase(s)>, Display app'"'"'s Docker options for phase (comma separated phase list)
     docker-options:add <app> <phase(s)> OPTION, Add Docker option to app for phase (comma separated phase list)
```

Comment:
ah sorry about that, it's because that's a separate issue about POSIX argument syntax standards compliance. We have something of a history of taking things into the weeds on minor points like that and I was hoping to not touch the topic (since that was preexisting before this PR).